app/section/sections/browser-additional-information/ui.test.js

```diff
@@ -0,0 +1,181 @@
+
+  const additionalAdvice = Selector('[data-test-id="section-additional-advice"]');
+
+  await t
+    .expect(additionalAdvice.innerText).eql('This information is diplayed on the Marketing page for buyers to view');
+});
+
+test('should render the solution summary question', async (t) => {
+  await pageSetup(t);
+
+  const summaryQuestion = Selector('[data-test-id="question-additional-information"]');
+
+  await t
+    .expect(summaryQuestion.find('label.nhsuk-label').innerText).eql('Do you have any additional browser based information (optional)')
+    .expect(summaryQuestion.find('span.nhsuk-hint').innerText).eql('Add any additional information or requirement your Solution needs to function')
+    .expect(summaryQuestion.find('textarea').count).eql(1)
+    .expect(summaryQuestion.find('[data-test-id="textarea-field-footer"]').innerText).eql('You have 500 characters remaining');
+});
+
+test('should populate the text fields with existing data', async (t) => {
+  pageSetup(t, true);
+
+  const theQuestions = Selector('form');
+
+  await t
+    .expect(theQuestions.find('[data-test-id="question-additional-information"]').find('textarea').value).eql('The solution additional information')
+});
+
+test('should render the submit button', async (t) => {
+  await pageSetup(t);
+
+  const submitButton = Selector('[data-test-id="section-submit-button"]');
+
+  await t
+    .expect(submitButton.find('button').count).eql(1);
+});
+
+test('should show error summary and validation for questions when they exceed the maxLength', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-additional-information')
+    .reply(400, {
+      maxLength: ['additional-information'],
+    });
+
+  const oneHundredCharacters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789';
+  const thousandCharacters = oneHundredCharacters.repeat(10);
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const additionalInformation = Selector('[data-test-id="question-additional-information"]');
+  
+  const submitButton = Selector('[data-test-id="section-submit-button"] button');
+  
+  await t
+  .expect(errorSummary.exists).notOk()
+  .typeText(additionalInformation, `${thousandCharacters}0`, { paste: true })
+  .click(submitButton)
+  .expect(errorSummary.exists).ok()
+  .expect(errorSummaryList.find('li').count).eql(1)
+  .expect(errorSummaryList.find('li:nth-child(1)').innerText).eql('Additional information is over the character limit')
+  .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#additional-information')
+  .expect(additionalInformation.find('.nhsuk-textarea--error').exists).ok()
+  .expect(additionalInformation.find('.nhsuk-error-message').innerText).eql('Error:\nAdditional information is over the character limit')

```

Comment:
can you indent all of these please. I'm not sure if your vscode has the eslint plugin. But you should be seeing red squigle lines to highlight this